### PR TITLE
NIRES dev

### PR DESCRIPTION
**Pull request by kbwestfall**
Makes some minor changes to one of the NIRES pypeit files and adds a test of the `pypeit_parse_calib_id` script.  Although I think that script probably should be deprecated.

This PR goes with https://github.com/pypeit/PypeIt/pull/1550 .

### CODE REVIEW

File: pypeit_files/keck_nires_nires.pypeit [hunk 1]
```diff
@@ -24,13 +24,13 @@ setup end
...
-| s190519_0067.fits | arc,science,tilt | 222.661173514297 | 33.0367669564831 | J1450+3302 |     spec | default |     1,1 | 58622.4110204323 | 1.03169892034606 |   300.0 |     1 |       3 |      4 |
-| s190519_0068.fits | arc,science,tilt | 222.660207102656 | 33.0382234687118 | J1450+3302 |     spec | default |     1,1 | 58622.4152114045 | 1.03446078772601 |   300.0 |     2 |       4 |      3 |
-| s190519_0010.fits |  pixelflat,trace |             58.0 |             45.0 |       flat |     spec | default |     1,1 | 58622.0743023767 | 1.41291034446565 |   100.0 |   all |      -1 |     -1 |
-| s190519_0012.fits |  pixelflat,trace |             58.0 |             45.0 |       flat |     spec | default |     1,1 | 58622.0769763351 | 1.41291034446565 |   100.0 |   all |      -1 |     -1 |
-| s190519_0014.fits |  pixelflat,trace |             58.0 |             45.0 |       flat |     spec | default |     1,1 | 58622.0796502934 | 1.41291034446565 |   100.0 |   all |      -1 |     -1 |
+           filename |         frametype |               ra |              dec |     target | dispname |  decker | binning |              mjd |          airmass | exptime | calib | comb_id | bkg_id
```
Comment (jhennawi) [inline pypeit_files/keck_nires_nires.pypeit:27]: We might want to add some comment here that if standards don't have enough exposure time they should not be used for tilts. I'm just thinking some users will look at this dev suite example and get confused. I think we discuss this in the A-B differencing docs, but would have to look.

File: pypeit_files/keck_nires_nires.pypeit [hunk 2]
```diff
@@ -24,13 +24,13 @@ setup end
...
-| s190519_0012.fits |  pixelflat,trace |             58.0 |             45.0 |       flat |     spec | default |     1,1 | 58622.0769763351 | 1.41291034446565 |   100.0 |   all |      -1 |     -1 |
-| s190519_0014.fits |  pixelflat,trace |             58.0 |             45.0 |       flat |     spec | default |     1,1 | 58622.0796502934 | 1.41291034446565 |   100.0 |   all |      -1 |     -1 |
+           filename |         frametype |               ra |              dec |     target | dispname |  decker | binning |              mjd |          airmass | exptime | calib | comb_id | bkg_id  
+  s190519_0059.fits | arc,standard,tilt | 194.259151362434 | 22.0313494322493 |      GD153 |     spec | default |     1,1 | 58622.3598610573 | 1.03675819208546 |   200.0 |     1 |       1 |      2  
+  s190519_0060.fits | arc,standard,tilt | 194.260349844733 | 22.0313316699865 |      GD153 |     spec | default |     1,1 |  58622.362605849 | 1.04142552296712 |   200.0 |     1 |       2 |      1  
+  s190519_0067.fits |  arc,science,tilt | 222.661173514297 | 33.0367669564831 | J1450+3302 |     spec | default |     1,1 | 58622.4110204323 | 1.03169892034606 |   300.0 |     2 |       3 |      4
```
Comment (jhennawi) [inline pypeit_files/keck_nires_nires.pypeit:30]: Now that we are fleshing this out more, I wonder if we should try to put the ABBA sequence for this data and perform the 3443 ABBA combine.

**Review by jhennawi — APPROVED** [review]
Minor comments.

File: pypeit_files/keck_nires_nires.pypeit (hunk 1 above)
Comment (kbwestfall) [reply pypeit_files/keck_nires_nires.pypeit:27]: Yes, we discuss this in the NIRES tutorial.

File: pypeit_files/keck_nires_nires.pypeit (hunk 2 above)
Comment (kbwestfall) [reply pypeit_files/keck_nires_nires.pypeit:30]: I'm going to punt on this for now.  We can update once we get the QL examples in place.

**Pull request merged**